tests/Integrations/PDO/PDOTest.php

```diff
@@ -72,7 +72,7 @@ public function testCustomPDOPrepareWithStringableStatement()
             )
                 ->setTraceAnalyticsCandidate()
                 ->withExactTags(array_merge($this->baseTags(), [
-                    'db.rowcount' => 1,
+                    'db.row_count' => 1,
```

Comment:
This PR is not consistent with `'db.row_count'` literal vs `Tag::DB_ROWCOUNT` constant.